python/lsst/qa/explorer/parquetTable.py

```diff
@@ -0,0 +1,268 @@
+"""
+Implementation of thin wrappers to pyarrow.ParquetFile.
+"""
+import re
+import json
+from itertools import product
+
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+
+class ParquetTable(object):
+    """Thin wrapper to pyarrow's ParquetFile object
+
+    Call `to_df` method to get a `pandas.DataFrame` object,
+    optionally passing specific columns.
+
+    Parameters
+    ----------
+    filename : str
+        Path to Parquet file.
+
+    """
+    def __init__(self, filename=None, dataFrame=None):
+        if filename is not None:
+            self._pf = pq.ParquetFile(filename)
+            self._df = None
+            self._pandas_md = None
+        elif dataFrame is not None:
+            self._df = dataFrame
+            self._pf = None
+        else:
+            raise ValueError('Either filename or dataFrame must be passed.')
+
+        self._columns = None
+        self._columnIndex = None  
+
+    @property
+    def df(self):
+        return self._df
+
+    @property
+    def pf(self):
+        return self._pf
+
+    def write(self, filename):
+        """Write pandas dataframe to parquet
+
+        Parameters
+        ----------
+        df : `pandas.DataFrame`
+            Dataframe to write to Parquet file.
+
+        filename : str
+            Path to which to write.
+        """
+        if self.df is None:
+            raise ValueError('df property must be defined to write.')
+        table = pa.Table.from_pandas(self.df)
+        pq.write_table(table, filename, compression='none')
+
+    @property
+    def pandas_md(self):
```

Comment:
Naming here and below is inconsistent between PEP8 and Science Pipelines style.  I think you're supposed to follow the latter (camelCase).